token.go

```diff
@@ -17,14 +17,40 @@ type TokenCVCUpdateParams struct {
 	CVC *string `form:"cvc"`
 }
 
+// TokenAccountParams is the set of parameters that can be used when creating a Account token.
+type TokenAccountParams struct {
+	BusinessType        *string               `form:"business_type"`
+	Company             *AccountCompanyParams `form:"company"`
+	Individual          *PersonParams         `form:"individual"`
+	TOSShownAndAccepted *bool                 `form:"tos_shown_and_accepted"`
+}
+
+// TokenRelationshipParams is the set of Relationship parameters that can be used inside TokenPersonParams.
+type TokenRelationshipParams struct {
+	Owner *bool `form:"owner"`
+}
+
+// TokenPersonParams is the set of parameters that can be used when creating a Person token.
+type TokenPersonParams struct {
+	FirstName    *string                  `form:"first_name"`
+	LastName     *string                  `form:"last_name"`
+	Relationship *TokenRelationshipParams `form:"relationship"`
```

Comment:
This has only 3 parameters but there are many more in the documentation: https://stripe.com/docs/api/tokens/create_person

Can you clarify why you choose only those 3? We should support all of them (or none) but I know it's a lot of work so I'm not requiring you to do it yourself.

The easiest (though not cleanest) would be to use `PersonParams` instead so removing those classes and replacing
```
Person      *TokenPersonParams    `form:"person"`
```
with 
```
Person      *PersonParams    `form:"person"`
```

Reply:
> Can you clarify why you choose only those 3? We should support all of them (or none)

I used this custom struct because I thought they were only 3 usable fields (those mentioned inside the documentation example).

It's a bad documentation reading 😆

> The easiest (though not cleanest) would be to use PersonParams instead so removing those classes and replacing

It's funny because I used this struct before creating the custom one.

> but I know it's a lot of work so I'm not requiring you to do it yourself.

Yeah, it's not a problem, I like contributing!
I will take a look tomorrow 😄

Reply:
@remi-stripe It should be ok!